src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -141,34 +156,36 @@ part of an escalation. To the extent possible, temporary delegation from
 For these reasons, the **Backup** is asked to at least maintain awareness of the
 issues the **Primary** is handling.
 
-In the ideal, the **Primary** is able to handle all customer response activity
+Ideally, the **Primary** is able to handle all SRE activity
 and no other developers are engaged. However, this situation can change for
-significant *escalations* (see below). Thus, other developers are free to ignore
-customer inquiries as well as redirect customers who may contact them directly
+significant :ref:`escalations <sre_escalations>`. Thus, other developers are free to
+ignore customer inquiries as well as redirect customers who may contact them directly
 via email, phone or walk-in. It is a best practice to handle such redirections
 with a formal, three-way handoff confirming that the customer indeed makes
 contact with the **Primary**.
 
-Active Customer Response Issues Repo
-------------------------------------
+.. _sre_active_issues:
 
-*Active* customer response issues will be logged and tracked in a separate GitHub,
+Active SRE Issues Repo
+----------------------
+
+*Active* SRE issues will be logged and tracked in a separate GitHub,
 `issues-only repository <https://github.com/visit-dav/live-customer-response/issues>`_
 within the `visit-dav GitHub organization <https://github.com/visit-dav>`_. 
 For each new inquiry, the primary will file an issue ticket and assign themselves.
-When the incident is resolved, the associated issue is closed. The primary will
+When the inquiry is resolved, the associated issue is closed. The primary will
```

Comment:
By "resolved" here, do we mean the resolution criteria in section 10.5? And if so, we should link to it here.